cmd/opentelemetry-collector/app/exporter/cassandra/factory.go

```diff
@@ -62,17 +64,25 @@ func (f Factory) CreateDefaultConfig() configmodels.Exporter {
 }
 
 // CreateTraceExporter creates Jaeger Cassandra trace exporter.
-// This function implements OTEL component.Factory interface.
-func (Factory) CreateTraceExporter(log *zap.Logger, cfg configmodels.Exporter) (component.TraceExporterOld, error) {
+// This function implements OTEL component.ExporterFactory interface.
+func (f Factory) CreateTraceExporter(
+	_ context.Context,
```

Comment:
why do they pass context? Is it used as a replacement for `Close()`?

Reply:
Close has its own context. This context seems to be associated with application start event.